src/ui/frames/admin_panel.py

```
@@ -13,57 +13,72 @@ def __init__(self, parent, view_model):
+        self.student_name_entry = ctk.CTkEntry(self.controls_frame, placeholder_text="Full Name (e.g., Le Nguyen Gia Hung)", font=ctk.CTkFont(family="Inter", size=14), corner_radius=8)
+        self.enroll_course_dropdown = ctk.CTkOptionMenu(self.controls_frame, variable=self.enroll_course_var, values=["Loading..."], command=self.on_enroll_course_selected, font=ctk.CTkFont(family="Inter", size=14), corner_radius=10)
+        self.enroll_class_dropdown = ctk.CTkOptionMenu(self.controls_frame, variable=self.enroll_class_var, values=["Select a course first"], font=ctk.CTkFont(family="Inter", size=14), corner_radius=10)
+        self.add_student_button = ctk.CTkButton(self.controls_frame, text="Add Student to Session", command=self.on_add_student_click, font=ctk.CTkFont(family="Poppins", size=14, weight="bold"), corner_radius=10)
+        self.session_list_frame = ctk.CTkScrollableFrame(self.controls_frame, height=100, label_text="Session Queue", corner_radius=10)
         
         self.retrain_button = ctk.CTkButton(
             self.controls_frame, 
             text="Re-Train Model", 
             fg_color="#3498DB", 
             hover_color="#2874A6",
             command=self.on_retrain_click, # Add the command
```

Flake8 (7.2.0)

[error] at least two spaces before inline comment

(E261)